doc/performance_blog_post/performance_blog_post.md

```diff
@@ -0,0 +1,109 @@
+# How we Improved the Files Virtual Schema Performance
+
+Our [Virtual Schemas for document files](https://github.com/exasol/virtual-schema-common-document-files) allow our customer to integrate data from files into their Exasol database. Customers use this feature for integrating external data on demand into their analytics and for building ETL pipelines.
+
+For Exasol speed matters. So does it for our integrations. For customers its crucial that the queries that include external data run as fast as possible. In this blog post I want to give some insight on the process of our recent performance improvements of the Virtual Schema for parquet files.
+
+## How it Begins
+
+For me this project started when our product manager asked me to improve the performance of the parquet file loading. He compared the performance of our products with competitors and found out, that parquet loading needs some improvements.
+
+In order to do professional performance improvements we use the simple workflow:
+
+Measure -> Improve -> Measure.
+
+In my eyes, this simple strategy brings lots ot advantages:  
+First it allows you to check weather a change helped or if it even made some functionality slower. For example improving the performance of loading big parquet files can easily slow down the loading of many small files. Second, this strategy allows you to define a goal when to stop. Otherwise, you will keep improving forever since there's always some space for improvements.
+
+## Writing Performance Tests
+
+So my first step was to write automated performance tests for the parquet loading. Don't get this wrong. It's not that we did not have any tests before. Of course, we had unit and integration tests for the Virtual Schema that make sure that the Virtual Schema works as expected. But since this is a rather new product we did not have automated performance tests yet.
```

Comment:
```suggestion
So my first step was to write automated performance tests for the Parquet loading. Don't get me wrong. It's not that we did not have any tests before. Of course, we had unit and integration tests for the Virtual Schema that makes sure that it works as expected. But since this is a rather new product we did not have automated performance tests yet.
```